.swiftformat

```diff
@@ -1,5 +1,20 @@
 #Disable all and explicitly enable them to avoid unintentionally activating new rules in newer versions of SwiftFormat
---disable all
+#--disable all
+
+--rules \
+	indent, \
+	duplicateImports, \
+	sortedImports, \
+	spaceInsideBraces, \
+	spaceInsideBrackets, \
+	spaceInsideComments, \
+	spaceInsideGenerics, \
+	spaceInsideParens, \
+	trailingCommas, \
```

Comment:
Would you remove `trailingCommas`? Seems to generate an unnecessary diff when we reformat.